### PR TITLE
Test for ChromeLoggerHandler

### PR DESCRIPTION
**Pull request by xiehappy**

### CODE REVIEW

File: system/Log/Handlers/ChromeLoggerHandler.php [hunk 1]
```diff
@@ -114,8 +114,10 @@ public function __construct(array $config = [])
 		$request = Services::request(null, true);
 
 		$this->json['request_uri'] = (string)$request->uri;
-
-		Events::on('post_controller', [$this, 'sendLogs'], HOOKS_PRIORITY_HIGH);
+                
+                Events::on('post_controller', [$this, 'sendLogs'], EVENT_PRIORITY_HIGH);
+                
+		//Events::on('post_controller', [$this, 'sendLogs'], HOOKS_PRIORITY_HIGH);
```
Comment (lonnieezell) [inline system/Log/Handlers/ChromeLoggerHandler.php]: The commented out line should be removed.

File: tests/system/HTTP/MessageTest.php [hunk 2]
```diff
@@ -198,6 +198,8 @@ public function testSetHeaderArrayValues()
       		
                 $this->assertEquals('json, html, xml', $this->message->getHeaderLine('Accept'));
         }
+        
+        //--------------------------------------------------------------------
```
Comment (lonnieezell) [inline tests/system/HTTP/MessageTest.php]: No need for a new separator here when there's one 2 lines below it.

File: tests/system/Log/ChromeLoggerHandlerTest.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+	//--------------------------------------------------------------------
+
+        public function testCanHandleLogLevel()
+	{
+		$config = new LoggerConfig();
+                $config->handlers['CodeIgniter\Log\Handlers\TestHandler']['handles'] =  ['critical'];
+		//$config->threshold = "foo";
+                
+		$logger = new ChromeLoggerHandler($config->handlers['CodeIgniter\Log\Handlers\TestHandler']);
```
Comment (lonnieezell) [inline tests/system/Log/ChromeLoggerHandlerTest.php]: Is this testing something specific to ChromeLoggerHandler or is this a feature of the BaseConfig? If so, it should be tested there.

File: system/Log/Handlers/ChromeLoggerHandler.php (hunk 1 above)
Comment (xiehappy) [inline system/Log/Handlers/ChromeLoggerHandler.php]: Yes. And I think "HOOKS_PRIORITY_HIGH" should be change to "EVENT_PRIORITY_HIGH". So I leave this comment here to let you check it.

Comment (lonnieezell) [inline system/Log/Handlers/ChromeLoggerHandler.php]: You're absolutely correct. It should be changed. And the constants themselves should be changed too. Thanks for catching that.

Comment (xiehappy) [inline system/Log/Handlers/ChromeLoggerHandler.php]: Thanks, I will change it.

File: tests/system/Log/ChromeLoggerHandlerTest.php (hunk 3 above)
Comment (xiehappy) [inline tests/system/Log/ChromeLoggerHandlerTest.php]: base config is an abstract class, and are using the chromelogger to test its functionality

Comment (lonnieezell) [inline tests/system/Log/ChromeLoggerHandlerTest.php]: Doh! Good point.

**Comment by jim-parry** [comment]
@lonnieezell This seems to be blowing up in tests/system/View/ParserPluginTest.php:29, which does not seem related to the PR??

**Pull request merged**